docs/ops.html

```diff
@@ -963,6 +963,21 @@ <h4><a id="remote_jmx" href="#remote_jmx">Security Considerations for Remote Mon
         <td>kafka.server:type=ReplicaManager,name=IsrExpandsPerSec</td>
         <td>See above</td>
       </tr>
+      <tr>
```

Comment:
maybe we can add the new metrics at end of the table?

Reply:
Well, I thought grouping `kafka.server:type=ReplicaManager`s would be better. Is there any policy?

Reply:
No there is policy, but `ReplicaManager`, `GroupMetadataManager` are not related, right? they can be separate group.